worker/changestream/worker.go

```diff
@@ -138,7 +142,7 @@ func (w *changeStreamWorker) GetWatchableDB(namespace string) (changestream.Watc
 		mux, err := w.cfg.NewWatchableDB(w.cfg.AgentTag, db, fileNotifyWatcher{
 			fileNotifier: w.cfg.FileNotifyWatcher,
 			fileName:     namespace,
-		}, w.cfg.Clock, w.cfg.Logger)
+		}, w.cfg.Clock, w.cfg.Metrics, w.cfg.Logger)
```

Comment:
This should pass a child of the metrics collector that has the namespace embedded. That way you don't need to pass around the namespace everywhere for the label names.

```
w.cfg.Metrics.ForNamespace(namespace)
```

Reply:
done